support-frontend/app/admin/settings/LandingPageTest.scala

```diff
@@ -32,13 +32,105 @@ object LandingPageCopy {
   implicit val codec: Codec[LandingPageCopy] = deriveCodec
 }
 
+case class ProductBenefit(
+    copy: String,
+    tooltip: Option[String] = None,
+    label: Option[Label] = None,
+)
+
+case class Label(
+    copy: String,
+)
+
+case class LandingPageProductDescription(
+    title: String,
+    label: Option[Label] = None,
+    benefits: List[ProductBenefit],
+    cta: Cta,
+)
+
+case class Cta(
+    copy: String,
+)
+
+case class Products(
+    Contribution: LandingPageProductDescription,
+    SupporterPlus: LandingPageProductDescription,
+    TierThree: LandingPageProductDescription,
+)
+
+object Products {
+  val defaultProducts: Products = Products(
```

Comment:
this will all come from the tool eventually, but hardcoded into the backend for now